Purchases/Misc/Obsoletions.swift

```diff
@@ -353,11 +387,16 @@ public extension Purchases {
      * - Parameter discount: The `SKProductDiscount` to apply to the product.
      * - Parameter product: The `SKProduct` the user intends to purchase.
      */
-    @available(iOS, introduced: 13.0, obsoleted: 13.0, message: "Obtain StoreProductDiscount from StoreProduct")
-    @available(tvOS, introduced: 13.0, obsoleted: 13.0, message: "Obtain StoreProductDiscount from StoreProduct")
-    @available(watchOS, introduced: 6.2, obsoleted: 6.2, message: "Obtain StoreProductDiscount from StoreProduct")
-    @available(macOS, introduced: 10.15, obsoleted: 10.15, message: "Obtain StoreProductDiscount from StoreProduct")
-    @available(macCatalyst, introduced: 13.0, obsoleted: 13.0, message: "Obtain StoreProductDiscount from StoreProduct")
+    @available(iOS, introduced: 13.0, obsoleted: 13.0,
+               message: "Obtain StoreProductDiscount from StoreProduct or use checkPromotionalOfferEligibility")
```

Comment:
feels like this method and its completion-block alternative should have the same message, right?

Reply:
Oh yeah, you right! Will fix